.goreleaser.yml

```diff
@@ -0,0 +1,47 @@
+project_name: fastly
+release:
+  prerelease: auto
+before:
+  hooks:
+    - go mod tidy
+    - go mod download
+builds:
+  - <<: &build_defaults
+      main: ./cmd/fastly
+      ldflags:
+        - -s -w -X "github.com/fastly/cli/pkg/version.AppVersion={{ .Version }}"
+        - -X "github.com/fastly/cli/pkg/version.GitRevision={{ .ShortCommit }}"
+        - -X "github.com/fastly/cli/pkg/version.GoVersion={{ .Env.GOVERSION }}"
```

Comment:
Are these standardized / defined somewhere else or are these parameters defined on these lines? Do you think it would be valuable to annotate the build number here so that we can find logs easier?

Edit: there's no such thing as a build number in GHA 😲

Reply:
These are defined and used inside the `fastly version` command and are used to populate the output. See here for more detail: https://github.com/fastly/cli/blob/master/pkg/version/root.go#L11-L67